src/misc/support.cpp

```diff
@@ -17,16 +17,15 @@
  */
 
 
-#include <string.h>
-#include <stdlib.h>
+#include <algorithm>
 #include <assert.h>
+#include <cctype>
 #include <ctype.h>
 #include <stdarg.h>
 #include <stdio.h>
-#include <string.h>
-#include <algorithm>
-#include <cctype>
+#include <stdlib.h>
 #include <string>
+#include <string.h>
```

Comment:
Let's convert it to `cstring` already, as it should've been done already by upstream…

Reply:
Yes, fixed in fb18d4e0